build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/QuarkusCliDevModeIT.java

```diff
@@ -24,20 +24,17 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.DisabledOnOs;
-import org.junit.jupiter.api.condition.OS;
 
 import io.quarkus.test.bootstrap.QuarkusCliRestService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.scenarios.annotations.DisabledOnNative;
 
-@DisabledOnOs(OS.WINDOWS) // TODO mvavrik: must only run when Quarkus CLI and Docker are available
 @Tag("QUARKUS-2812")
 @Tag("quarkus-cli")
 @QuarkusScenario
 @DisabledOnNative(reason = "Only for JVM verification")
-public class DevModeIT extends AbstractAnalyticsIT {
+public class QuarkusCliDevModeIT extends AbstractAnalyticsIT {
```

Comment:
Regarding renaming in general: why `@Tag("quarkus-cli")` is not enough?

Reply:
It's not necessary, but shouldn't QuarkusCLi tests start with QuarkusCli? If you think about it, then `DevModeIT` really tells you nothing. You can look at packages for context, but it's not how we usually name CLI tests, no?

Anyway, from POV of my concern - detecting CLI test - the tag is just enough, I just didn't realize it during implementation (should have you as a reviewer :-)), so if you don't want to rename it, I'll annotate the test for now. @fedinskiy please state your final thoughts

Reply:
btw from packages or name it is not clear we use Quarkus CLI and therefore it won't be clear in tests results in Jenkins unless you check logs.

Reply:
I am strongly against using names for detection purposes (we already have this for Openshift, and it creates to much of visual clutter like `OpenShiftBinaryCloudEventsOverProdAmqpIT` and `OpenShiftPostgresqlTransactionGeneralUsageIT`), but I am fine with changing names for readability purposes. I suggest to change 'QuarkusCliSomethingIT' into 'CliSomethingIt', but that is up to you.

Reply:
I'm creating PR for FW to handle this, but basically my new approach is: use the annotation on all the CLI tests that requires Docker (as this way, we have chance to run all CLI tests that doesn't need it on Windows Github CI). Then I need to go step by step and determine when Docker is in action on CLI tests.